PMM-8982: Migrate unfinished logs

oter opened the pull request:
[PMM-8982](https://jira.percona.com/browse/PMM-8982)

- [x] Build: https://github.com/Percona-Lab/pmm-submodules/pull/2119

- [x] Links to other linked pull requests (optional).
---
- [x] Tests passed.
- [x] Feature build pass.
- [x] (Re)requested review.
- [x] Fix conflicts with target branch.

BupycHuk reviewed (approved):
looks good to me, what about cases when pmm-agent has outdated version?

oter commented:
> looks good to me, what about cases when pmm-agent has outdated version?

The outdated pmm-agent should not affect the bug I fixed. My fix is about the logs functionality implementation which is not present in the previous version of pmm-managed and added in the following version, and we forgot to mark old jobs that they have no logs at all.
